### DISCUSSION TITLE
Update record with multi step form

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7627] [Help] [answered]
I have a long form with 10 form steps.
All these steps data are saved into one table record in DB.
The record is saved at each step , including the last completed step number.

When I come back to an existing record, I'd like to change the current form step immediately, to go to the last completed step, to avoid re-validate each step.

I did not find a condition to do it...

CC @mielune

**Comment by melohagan** [comment, marked as answer]
This would definitely be good to add. There's an open issue here: https://github.com/Budibase/budibase/issues/7063
